api/comment_threads.rb

```diff
@@ -78,6 +78,7 @@
   comment.anonymous_to_peers = bool_anonymous_to_peers || false
   comment.author = user
   comment.comment_thread = thread
+  comment.children_count = 0
```

Comment:
@robrap The name of the attribute which stores the nested items is `children`. So `children_count` seems more consistent.